src/models/applicationState.ts

```diff
@@ -98,6 +98,7 @@ export interface IAppSettings {
 export interface IProject {
```

Comment:
What do you think about defining the `version` as optional - So it makes defining the objects easier from the consumer side and then the services/reducers job is to set it when it changes.

Reply:
It's "optional" to read of course but I think it should be mandatory to always persist project (and asset) with this new value.  If we mark this optional we're not guaranteed.